### PR TITLE
switch from exp/slices pkg to the stable

### PR DESCRIPTION
**Pull request by poopoothegorilla**
Since we have upgraded our go version to 1.21.1 we should migrate from the exp/slices pkg to the now stable slices pkg which was recently merged into the standard go project as of 1.21.1 https://pkg.go.dev/slices

This will also help with our future dependency upgrades such as upgrading the geth [version](https://github.com/smartcontractkit/chainlink/pull/10264/commits/f92ae4523eb4b31d508e6b64f7735a43162ba65d)

Summary:
- changed references to `exp/slices` pkg to use the standard libraries stable `slices` pkg
- changed functions which needed to be updated for compatibility with the new pkg signature
- changed: core/services/vrf/v2/listener_v2.go
- changed: core/chains/cosmos/cosmostxm/txm.go
- changed: core/chains/evm/config/toml/defaults.go

### CODE REVIEW

**Review by amit-momin — DISMISSED** [review]
LGTM! I don't have merge privileges though so we'll still have to wait on someone else to look it over and merge it

File: core/services/vrf/v2/listener_v2.go [hunk 1]
```diff
@@ -503,8 +503,14 @@ func (lsn *listenerV2) processPendingVRFRequests(ctx context.Context) {
...
-		slices.SortFunc(reqs, func(a, b pendingRequest) bool {
-			return a.req.CallbackGasLimit() < b.req.CallbackGasLimit()
+		slices.SortFunc(reqs, func(a, b pendingRequest) int {
+			if a.req.CallbackGasLimit() == b.req.CallbackGasLimit() {
+				return 0
+			}
+			if a.req.CallbackGasLimit() < b.req.CallbackGasLimit() {
+				return -1
+			}
+			return 1 // a.req.CallbackGasLimit() > b.req.CallbackGasLimit()
```
Comment (jmank88) [inline core/services/vrf/v2/listener_v2.go:513]: Could this be reduced to a subtraction?

Comment (makramkd) [reply core/services/vrf/v2/listener_v2.go:513]: I don't think so, these are unsigned integers so there will be overflow.

**Review by prashantkumar1982 — APPROVED** [review]

File: core/services/vrf/v2/listener_v2.go (hunk 1 above)
Comment (jmank88) [reply core/services/vrf/v2/listener_v2.go:513]: Oh, what about:
```suggestion
			return cmp.Compare(a.req.CallbackGasLimit(), b.req.CallbackGasLimit())
```

File: core/chains/cosmos/cosmostxm/txm.go [hunk 2]
```diff
@@ -176,12 +177,21 @@ func (e *msgValidator) add(msg adapters.Msg) {
...
 		ac, bc := a.CreatedAt, b.CreatedAt
 		if ac.Equal(bc) {
-			return a.ID < b.ID
+			if a.ID < b.ID {
+				return -1
+			}
+			if a.ID > b.ID {
+				return 1
+			}
+			return 0
```
Comment (jmank88) [inline core/chains/cosmos/cosmostxm/txm.go:189]:
```suggestion
			return cmp.Compare(a.ID, b.ID)
```

File: core/services/vrf/v2/listener_v2.go (hunk 1 above)
Comment (makramkd) [reply core/services/vrf/v2/listener_v2.go:513]: Yup that looks cleaner 👍

**Review by prashantkumar1982 — DISMISSED** [review]

**Review by jmank88 — DISMISSED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**